.devcontainer/devcontainer.json

```diff
@@ -1,14 +1,12 @@
 {
     "name": "TypeScriptToLua",
-    "image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:14",
-    "settings": {
-        "terminal.integrated.shell.linux": "/bin/bash"
-    },
+    "image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:18",
     "extensions": [
         "typescript-to-lua.vscode-typescript-to-lua",
         "dbaeumer.vscode-eslint",
         "editorconfig.editorconfig",
         "esbenp.prettier-vscode"
     ],
-    "postCreateCommand": "npm ci"
+    "postCreateCommand": "npm ci --legacy-peer-deps",
```

Comment:
legacy-peer-deps should not be in here, this is something I'm looking to fix, not something we want to keep

Reply:
Ok, no problem to remove it. The post-create command failing in the meantime is a very minor inconvenience.